### DISCUSSION TITLE
Unable to convert to Visio file from draw.io

### DISCUSSION THREAD

**Discussion by pranjalparitosh** [jgraph/drawio-desktop#1440] [General]
The complex architect flows in the Genesys cloud are not being converted to the visio file from the draw.io.
However, small flows are successfully converted to the visio file without any issue.
We tried a couple of version of draw.IO to make it work but nothing worked.
Also,for **complex flows** even draw.IO file was not being converted and we tried the following steps to make it work:

Select [Arrange]->[Insert]->[Advanced]->[CSV]
Delete sample settings
Paste this flow's csv file
Select Import

By doing this, the draw.IO file is created successfully for the complex flows but further creates partial flowchart in the visio file.
Please help us in resolving the two issues:

1- It creates draw.IO file automatically for complex flows without manually importing it using the above mentioned steps.
2- Once the draw.IO file is created, it can successfully create visio file (.vsdx).

**Comment by pranjalparitosh** [comment]
The callflows which are complex are getting exported to the draw.io file but the content inside it is blank.
We get an error pop-up while opening the file. Please find the error message displayed below:
"**Error Loading file, Not a diagram File (error on line 8547 at column 360: specification mandates value for attribute refs)**"

I also tried by updating the draw.io to the latest version and it's the same error for the complex flows.
We are having a discussion for the same on the community, please find the link below to refer to the updates:
[https://community.genesys.com/discussion/creates-flowcharts-of-genesys-cloud-or-pureconnect-interaction-flow?ReturnUrl=%2fcommunities%2fcommunity-home1%2fdigestviewer%3fCommunityKey%3dbab95e9c-6bbe-4a13-8ade-8ec0faf733d4].

We need your help to resolve this issue at the earliest!

**Comment by davidjgraph** [comment]
I don't understand the report. If you have a bug, submit it as an issue as a properly formatted bug report please.
